README.md

```diff
@@ -203,5 +203,5 @@ We are working on a more seamless way to run Kontain on MacOS and Windows.
 Copyright © 2021 Kontain Inc. All rights reserved.
 
 <a class="anchor" id="1">🔗</a>
-kontainapp/km is licensed under the [Apache 2.0 license](https://github.com/kontainapp/km/blob/master/LICENSE).
+kontainapp/km is licensed under the [Apache 2.0 license](LICENSE).
 Stay tuned.
```

Comment:
Why are we saying to "Stay tuned"?  For what?

Perhaps the answer is that, before we accept contributions of code, we require a contributor license agreement (CLA).  Jono pointed me to https://easycla.lfx.linuxfoundation.org/#/ which we should look into.  In the mean time, perhaps instead of "Stay tuned" we should say "Before you can contribute code, you will need to execute a Contributor License Agreement.  We haven't yet implemented an automated CLA system, so in the meantime, please email us if you'd like to contribute."

Reply:
For the record, I think 'Stay tuned' was linked to 'We are working on a more seamless way to run Kontain on MacOS and Windows'. In that context, it's redundant.  
If you want to add this caveat about contributing code, it should go in the 'Contributing' section in this README. I suggest adding this section from the README to the beginning of the CONTRIBUTING.md file.